docs/srt-live-transmit.md

```diff
@@ -267,6 +267,7 @@ shell (using **"** **"** quotes or backslash).
     - **The timeout mechanism doesn't work on Windows at all.** It behaves as if the timeout was set to **-1** and it's not modifiable.
 - **-timeout-mode, -tm** - Timeout mode used. Default is 0 - timeout will happen after the specified time. Mode 1 cancels the timeout if the connection was established.
 - **-st, -srctime, -sourcetime** - Enable source time passthrough. Default: disabled. It is recommended to build SRT with monotonic (`-DENABLE_MONOTONIC_CLOCK=ON`) or C++ 11 steady (`-DENABLE_STDCXX_SYNC=ON`) clock to use this feature.
+- **-buffering** - Enable source buffering up to the specified number of packets. Default: 10. Minimum: 1.
```

Comment:
Is the `1` a special value that means "no buffering"? Or the architecture doesn't allow for direct passthru anyway? If so, I think it should be mentioned, as most of the users will likely use this option to turn off buffering.

Reply:
Yes, the value of `1` means one packet is read from source, and passed to target.
Changing to:
> Minimum: 1 (no buffering).

Reply:
Fair enough.